plugin.php

```diff
@@ -169,6 +169,7 @@
 			$pPdfDocumentModel = new PdfDocumentModel($pWP->query_vars['estate_id'] ?? 0, $pWP->query_vars['view'] ?? '');
 			/* @var $pPdfDownload PdfDownload */
 			$pPdfDownload = $pDI->get(PdfDownload::class);
+            $pPdfDownload->settingGoogleBotAcceptIndex(!empty(get_option('onoffice-settings-google-bot-index-pdf-expose')));
```

Comment:
`PdfDownload` should read that option, so that reading the option can be tested as well.

Reply:
I updated